pages/parentenrollclass.tsx

```diff
@@ -0,0 +1,142 @@
+import Wrapper from "@components/SDCWrapper";
+import {
+    Link,
+    Box,
+    Button,
+    Stack,
+    Center,
+    Text,
+    useRadio,
+    useRadioGroup,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import { CloseIcon } from "@chakra-ui/icons";
+
+export default function ParentEnrollClass(): JSX.Element {
+    // Started creating buttons, found a thread that used radio buttons instead
+    // const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+    // const registerChildren = [];
+
+    // for (const child of children) {
+    //     registerChildren.push(
+    //         <Center>
+    //             <Button
+    //                 mb="25px"
+    //                 height="50px"
+    //                 width="340px"
+    //                 borderRadius="6px"
+    //                 textColor="#0C53A0"
+    //                 fontSize="16px"
+    //                 fontWeight="normal"
+    //                 backgroundColor="white"
+    //                 border="2px solid #E1E1E1"
+    //             >
+    //                 {child}
+    //             </Button>
+    //         </Center>,
+    //     );
+    // }
+    const [isDisabled, setIsDisabled] = useState<boolean>(true);
+    function RadioCard(props) {
+        const { getInputProps, getCheckboxProps } = useRadio(props);
+
+        const input = getInputProps();
+        const checkbox = getCheckboxProps();
+
+        return (
+            <Box as="label">
+                <input {...input} />
+                <Box
+                    {...checkbox}
+                    lineHeight="24px"
+                    fontSize="16px"
+                    fontWeight="normal"
+                    textColor="#0C53A0"
+                    width="340px"
+                    height="50px"
+                    border="2px solid #E1E1E1"
+                    borderRadius="6px"
+                    _checked={{
+                        fontWeight: "normal",
+                        bg: "white",
+                        textColor: "#0C53A0",
+                        border: "2px solid #0C53A0",
+                    }}
+                    textAlign="center"
+                    py="13px"
+                    my="10px"
+                >
+                    {props.children}
+                </Box>
+            </Box>
+        );
+    }
+    function ChildrenButtons() {
+        // children to be retrieved from parent enroll endpoint
+        const children = ["Christina Ru", "Raewyn Tsai", "Stacy Kwok"];
+
+        const { getRootProps, getRadioProps } = useRadioGroup({
+            name: "framework",
+            defaultValue: "react",
+            onChange: () => setIsDisabled(false),
+        });
+
+        const group = getRootProps();
```

Comment:
What's getRootProps() doing?

Reply:
I'm not exactly sure, I took some code from the chakra ui docs but I think it gives the corresponding component some default props, I console logged the `group` prop getter for the `Stack` component and took a screenshot
![image](https://user-images.githubusercontent.com/71860803/127939703-ca9f7b6d-cca6-40f7-a9b5-259bbee2ab57.png)